settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
-    gradle.ext.kotlinVersion = '1.9.22'
-    gradle.ext.kspVersion = '1.9.22-1.0.17'
+    gradle.ext.kotlinVersion = '1.9.24'
```

Comment:
Do we really want to update the `kotlinVersion` and `kspVersion` in this PR? That seems like something that should be handled separately.

Reply:
This was bumped to align with the https://github.com/automattic/wordpress-rs project,t which used very slightly newer versions